engines/scumm/imuse_digi/dimuse_codecs.cpp

```diff
@@ -287,7 +287,11 @@ int32 decompressADPCM(byte *compInput, byte *compOutput, int channels) {
 			// Clip outputWord to 16 bit signed, and write it into the destination stream
 			outputWord = CLIP<int32>(outputWord, -0x8000, 0x7fff);
 
-			WRITE_LE_UINT16(dst + destPos, outputWord);
+			// This is being written as-is (LE), without concerns regarding endianness:
+			// this is because the internal DiMUSE mixer handles the data in LE format,
+			// and we'll convert data to the appropriate format using the QueuingAudioStream flags
+			// when flushing the final audio data to the output stream (see dimuse_waveout.cpp:53)
```

Comment:
don't add line numbers, refer to function name or something

Reply:
Thanks